python/coinbase-agentkit/README.md

```diff
@@ -314,6 +335,42 @@ agent_kit = AgentKit(AgentKitConfig(
 ))
 ```
 
+#### Configuring `EthAccountWalletProvider` gas parameters
+
+The `EthAccountWalletProvider` also exposes parameters for effecting the gas calculations.
+
+```python
+from eth_account import Account
+
+from coinbase_agentkit import (
+    AgentKit, 
+    AgentKitConfig, 
+    EthAccountWalletProvider, 
+    EthAccountWalletProviderConfig
+)
+
+private_key = os.environ.get("PRIVATE_KEY")
```

Comment:
Please add the import for `import os` to the top of these 2 `EthAccountWalletProvider` code snippets